testing-proto/src/main/proto/io/grpc/testing/integration/test.proto

```diff
@@ -47,6 +47,11 @@ service TestService {
   // One request followed by one response.
   rpc UnaryCall(SimpleRequest) returns (SimpleResponse);
 
+  // One request followed by one response. Response has cache control
```

Comment:
Normally it is good to briefly sum up the important parts from the spec for the service implementation. Namely, that the request is ignored and the response is always different, except for caching based on the request.